docs/getting-started/test-verification-server.md

```diff
@@ -3,10 +3,28 @@
 A basic eligibility verification server is available for testing:
 
 ```bash
-docker-compose up --build server
+docker compose up [-d] server
 ```
 
-The API endpoint is running at `http://localhost:5000/verify`.
+The optional `-d` flag will start in _detatched_ mode and allow you to continue using the terminal session. Otherwise your
+terminal will be attached to the container's terminal, showing the startup and runtime output.
 
-From within a local `client` container, the API endpoint is at `http://server:5000/verify`
-using the service-forwarding features of Docker Compose.
+The API server is running on `http://localhost` at a port dynamically assigned by Docker. To find the exact address, look in
+the Docker Desktop UI or run the following command:
+
+```bash
+docker ps -f name=server
+```
+
+The `PORTS` column should look like:
+
+```console
+0.0.0.0:$LOCAL_PORT->5000/tcp, 0.0.0.0:$DEBUG_PORT->5678/tcp
+```
+
+Where `$LOCAL_PORT` and `$DEBUG_PORT` are the dynamically assigned `localhost` ports.
+
+From within another Compose service container, the API is at `http://server:5000` using the service-forwarding features of
+Compose.
+
+In either case, the endpoint `/verify` serves as the Eligibility Verification API endpoint.
```

Comment:
- I am able to successfully build and run `docker compose build server`, `docker compose up server`

- Note: This log might be confusing to users because the `* Running on http://172.26.0.3:5000/` is now incorrect
![image](https://user-images.githubusercontent.com/3673236/135666236-602ee738-25fb-4a29-a9d2-67a259660a24.png)

- When I run the Docker app and the `docker ps` command, I see that the port is assigned to `57463`:
![image](https://user-images.githubusercontent.com/3673236/135666176-85f762e4-bd5a-4ba1-8b5b-87f75c65d3c2.png)
![image](https://user-images.githubusercontent.com/3673236/135666404-51624bf5-d747-4f9f-aa9f-9017a370c8cf.png)

- When I have both the server and client running via docker-compose, I am able to go through a successful verification flow:
- 
![image](https://user-images.githubusercontent.com/3673236/135666707-4f69a5fa-9b6d-4cc9-b369-d6fa5c8148a2.png)
![image](https://user-images.githubusercontent.com/3673236/135666723-128a7629-e1a9-45d3-9ac0-8940c65f0957.png)